### PR TITLE
Loading

### PR DESCRIPTION
**Pull request by lightme-fan**
Hey everyone,

I have just finished working on the loading or pending indicator. Please have a look at it.

Thanks in advance

- Branch: [https://github.com/onja-org/vamalaka-fannar/tree/loading](https://github.com/onja-org/vamalaka-fannar/tree/loading)
- ClickUp: [https://app.clickup.com/t/45e3jw](https://app.clickup.com/t/45e3jw)
- Refs: #45e3jw

### CODE REVIEW

File: src/components/Loading/Loading.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react'
+import styled from 'styled-components'
+
+export interface SourceType {
+  source: any
```
Comment (Tiaraha-funny) [inline src/components/Loading/Loading.tsx:5]: Maybe we shouldn't have "any" type

File: src/components/Loading/Loading.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      -webkit-transform: rotate(360deg);
+      transform: rotate(360deg);
+    }
+  }
+  img {
+    -webkit-animation: spin 2s linear infinite;
+    -moz-animation: spin 2s linear infinite;
+    animation: spin 2s linear infinite;
+  }
+`
```
Comment (andrzej-onja) [inline src/components/Loading/Loading.tsx:38]: have you used svg from the clickup ticket attachment ? it was already animating

File: src/components/Loading/icons/extremely-samll.svg [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (andrzej-onja) [inline src/components/Loading/icons/extremely-samll.svg:1]: please use one react component for all sizes that wraps svg and passes dimensions as a prop

File: src/components/Loading/Loading.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react'
+import styled from 'styled-components'
+
+export interface SourceType {
+  source: any
+}
+
+export const Loading: FC<SourceType> = ({ source }) => {
```
Comment (andrzej-onja) [inline src/components/Loading/Loading.tsx:8]: rename to LoadingIndicator
- remove source prop and create size  prop
 size: 'xl'| "lg"  etc. similar to what we have in mediaqueries.ts

**Review by andrzej-onja — CHANGES_REQUESTED** [review]

**Review by andrzej-onja — APPROVED** [review]

**Review by Tiaraha-funny — APPROVED** [review]

**Review by mitia-anah — APPROVED** [review]

**Pull request merged**